### ISSUE TITLE
pgadmin4 doesn't work with psql 14/pgo 5

### ISSUE THREAD

**Issue by tirelibirefe** [CrunchyData/postgres-operator#3375]
Hello,
pgo 5.1
psql 14
eks 1.2

I followed the instructions [here](https://access.crunchydata.com/documentation/postgres-operator/5.2.0/architecture/pgadmin4/):

```
  userInterface:
    pgAdmin:
      image: registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi8-4.30-3
      dataVolumeClaimSpec:
        accessModes:
        - "ReadWriteOnce"
        resources:
          requests:
            storage: 1Gi
```

Instead of port-forward, I prepared an ingress to connect ui.

pgadmin works, can be logged on but pgadmin cannot connect to the db instance. I get the following error message:

`'psycopg2.extensions.Column' object has no attribute '_asdict'`

![image](https://user-images.githubusercontent.com/45227339/188883787-bc1e3eeb-4e62-4033-a562-fdbbbc0276ee.png)

Thanks & Regards

**Comment by loxy** [comment]
See: https://github.com/CrunchyData/postgres-operator/issues/3367

**Comment by cbandy** [comment]
This is a bug in our build process. You can downgrade to tag `ubi8-4.30-2` until it is fixed.

**Comment by tjmoore4** [comment]
@tirelibirefe This has been addressed with [crunchydata/crunchy-containers#1480](https://github.com/CrunchyData/crunchy-containers/pull/1480) and is planned for inclusion in the next release.
